script.js

```diff
@@ -0,0 +1,83 @@
+// 1 задание
+
+const loop = (times = 0, callback = null) => {
+    if (callback != null) {
+        for (i = 0; i < times; i++) {
```

Comment:
```suggestion
        for (let i = 0; i < times; i++) {
```
в противном случае засоряем глобальный объект или же получаем ошибку при 'strict mode' :)